exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,87 @@
+{
+  "exercise": "armstrong-numbers",
+  "version": "1.0.0",
+  "description": "There are no 2 digit Armstrong numbers",
```

Comment:
I don't think this line is needed.  It is also causing Travis CI to fail as it violates the [canonical-schema.json](https://github.com/exercism/problem-specifications/blob/master/canonical-schema.json)

Reply:
This should be the description for a test of a 2 digit number.

Reply:
Wouldn't it still be `1.0.0`?

Reply:
The canonical-data `version` key is correct.

It was an out of date "commit" I was looking at.
Confusing word choice by me there sorry.